librz/core/cmd/cmd_analysis.c

```diff
@@ -557,18 +557,13 @@ static void list_vars(RzCore *core, RzAnalysisFunction *fcn, PJ *pj, int type, c
 	}
 }
 
-static int cmd_an(RzCore *core, bool use_json, const char *name) {
-	int ret = 0;
+static int core_analysis_name_json(RzCore *core, const char *name, PJ *pj) {
```

Comment:
this function could be easily merged with `core_analysis_name` by just replacing the printing part of it.